integrations/dalle/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dalle",
+  "scripts": {
+    "type:check": "tsc --noEmit",
+    "version": "bp --version",
+    "login": "bp login",
+    "dev": "bp dev",
+    "gen": "bp gen",
+    "deploy": "bp deploy"
+  },
+  "keywords": [],
+  "private": true,
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@botpress/client": "0.10.1",
```

Comment:
**chore:** since the integration will now live in the `botpress/botpress` monrepo, the client and SDK version should both point to the local workspace:
```json
{
  "@botpress/client": "workspace:*",
  "@botpress/sdk": "workspace:*",
}
```